queries.yaml

```diff
@@ -1740,6 +1741,58 @@ post_process:
       source_layer: buildings
       target_value_type: int
 
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: buildings
+      start_zoom: 0
+      end_zoom: 17
+      properties:
+        - landuse_kind
+      where: >-
+        landuse_kind not in ['aerodrome', 'airfield', 'cemetery', 'college', 'hospital', 'park', 'school', 'university']
```

Comment:
Looking at Bubble Wrap there are a few more we should add: 

- runway, taxiway, national_park, forest, nature_reserve, protected_area, conservation, golf_course, garden